src/FastRoute/src/GlobalRouter.cpp

```diff
@@ -121,38 +121,23 @@ void GlobalRouter::deleteComponents()
   delete _openSta;
   delete _result;
   delete _routingLayers;
-  
-  delete _block;
-
 }
 
-void GlobalRouter::resetResources()
+void GlobalRouter::clear()
 {
-  deleteComponents();
-
-  _vCapacities.clear();
-  _hCapacities.clear();
-
-  makeComponents();
-}
-
-void GlobalRouter::reset()
-{
-  deleteComponents();
+  // Clear classes
+  _grid->clear();
+  delete _fastRoute;
+  _fastRoute = new FT;
```

Comment:
I'm not sure if this is the best way to clear _fastRoute object. I deleted it since it has a lot of pointers inside of it. @jjcherry56 do you have any comment about it?

Reply:
update: added clear() function to FastRoute core class